crates/volta-core/src/tool/mod.rs

```diff
@@ -31,7 +31,7 @@ fn debug_already_fetched<T: Display + Sized>(tool: T) {
 
 #[inline]
 fn info_installed<T: Display + Sized>(tool: T) {
-    info!("{} installed and set {} as default", success_prefix(), tool);
+    info!("{} installed and set {} as default\n{} to use {tool} in this project run `volta pin {too;
```

Comment:
Looks like this is missing some of the necessary bits to actually present the right message here!

```suggestion
    info!("{} installed and set {} as default", success_prefix(), tool);
    info!("{} to use {} in this project run `volta pin {}`, success_prefix(), tool);
```